src/components/Settings/UsageReportSettings.tsx

```diff
@@ -0,0 +1,40 @@
+import { Flex, Text, Checkbox, Link } from '@radix-ui/themes'
+import { Controller, useFormContext } from 'react-hook-form'
+import { SettingsSection } from './SettingsSection'
+import { AppSettings } from '@/types/settings'
+
+export const UsageReportSettings = () => {
+  const { control, register } = useFormContext<AppSettings>()
+
+  const handleLinkClick = () => {
+    window.studio.browser.openExternalLink(
+      'https://github.com/grafana/k6-studio/blob/main/README.md#usage-report'
+    )
+  }
+
+  return (
+    <SettingsSection title="Usage Report">
+      <Flex gap="2">
+        <Controller
+          control={control}
+          name="usageReport.enabled"
+          render={({ field }) => (
+            <Text size="2" as="label">
+              <Checkbox
+                {...register('usageReport.enabled')}
+                defaultChecked
```

Comment:
IMO this is redundant: the field should reflect the current settings: if they are saved in the file, the user will see the value from the file; if not, the user will see the default value